cfy_manager/components/base_component/base_component.py

```diff
@@ -13,10 +13,16 @@
 #  * See the License for the specific language governing permissions and
 #  * limitations under the License.
 
+import os
+
+from ...config import config
```

Comment:
Import styling: top to bottom - should be 1 dot then 2 dots then 3...etc...

Reply:
Fixed.